test/int/wizards/breakpoints/implementation/breakpointsAssertions.ts

```diff
@@ -29,11 +34,43 @@ interface IObjectWithLocation {
 export class BreakpointsAssertions {
     private readonly _variableAssertions = new VariablesAssertions(this._internal.client);
 
+    private readonly _defaultStackFrameFormat: DebugProtocol.StackFrameFormat = {
+        parameters: true,
+        parameterTypes: true,
+        parameterNames: true,
+        line: true,
+        module: true
+    };
+
     public constructor(
         private readonly _breakpointsWizard: BreakpointsWizard,
         private readonly _internal: InternalFileBreakpointsWizard,
         public readonly currentBreakpointsMapping: CurrentBreakpointsMapping) { }
 
+    private getDefaultStackTraceVerifier(breakpoint: BreakpointWizard, expectedStackTrace: string): IStackTraceVerifier {
+        return {
+            format: this._defaultStackFrameFormat,
+            verify: (stackTraceResponse) => {
+                expect(stackTraceResponse.success, `Expected the response to the stack trace request to be succesful yet it failed: ${JSON.stringify(stackTraceResponse)}`).to.equal(true);
+                expect(stackTraceResponse.body.totalFrames, `The number of stackFrames was different than the value supplied on the totalFrames field`)
+                    .to.equal(stackTraceResponse.body.stackFrames.length);
+                stackTraceResponse.body.stackFrames.forEach(frame => {
+                    // Warning: We don't currently validate frame.source.path
+                    expect(frame.source).not.to.equal(undefined);
+                    const expectedSourceNameAndLine = ` [${frame.source!.name}] Line ${frame.line}`;
+                    expect(frame.name, 'Expected the formatted name to match the source name and line supplied as individual attributes').to.endsWith(expectedSourceNameAndLine);
+                });
+
+                const stackTrace = stackTraceResponse.body.stackFrames;
```

Comment:
If you move this up to line 54, you can use it for line 56 and 57

Reply:
Fixed